cli/replicasets/common.go

```diff
@@ -176,3 +176,12 @@ func connectToInstance(instanceName string, ctx *context.Ctx) (net.Conn, error)
 
 	return conn, nil
 }
+
+func needToWaitForHealthy(conn net.Conn) (bool, error) {
```

Comment:
`needToWaitForHealthy ` calls `GetMajorCartridgeVersion `. Is it mistake?

Reply:
No, I was chosen between this approach and calling `GetMajorCartridgeVersion` in each place `editReplicasets` is called.